ci-operator/config/openshift/ci-search/openshift-ci-search-master.yaml

```diff
@@ -3,21 +3,21 @@ base_images:
     name: centos
     namespace: openshift
     tag: "7"
-  openshift_release_golang-1.13:
+  openshift_release_golang-1.17:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.17
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.17
 images:
 - from: base
   inputs:
-    openshift_release_golang-1.13:
+    openshift_release_golang-1.17:
       as:
-      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.17
```

Comment:
This needs to be: registry.ci.openshift.org/openshift/release:golang-1.17

Reply:
Sounds good, updated